src/Reflection/SignatureMap/functionMap.php

```diff
@@ -404,7 +404,7 @@
 'BadMethodCallException::getPrevious' => ['?Throwable|?BadMethodCallException'],
 'BadMethodCallException::getTrace' => ['array'],
 'BadMethodCallException::getTraceAsString' => ['string'],
-'base64_decode' => ['string|false', 'str'=>'string', 'strict='=>'bool'],
+'base64_decode' => ['string|false', 'str'=>'string', 'strict='=>'bool|false'],
```

Comment:
@ohartl 
False is subtype of bool. Is it necessary to add it there?

Reply:
It might not be necessary. I added the default value `false` because other functions in the map do it too (Solr stuff, ..) . So I guess i will remove it again.

Reply:
Reverted the change in 49cd3a0